### ISSUE TITLE
Not Fullscreen in portrait mode

### ISSUE THREAD

**Issue by CHENHUI-XU** [jhomlala/betterplayer#545]
**History check**
Please confirm that you've checked issues history and you didn't find anything which may solve your issue.
Checked.

**Describe the bug**
When opening the player in fullscreen mode of portrait mode, the video is not technically displayed as fullscreen.

**To Reproduce**
Open portrait video in fullscreen
(even with aspectratio indicated as 16/9 or 9/16)

**Example code*
````
BetterPlayerDataSource betterPlayerDataSource = BetterPlayerDataSource(
      BetterPlayerDataSourceType.network,
      video,
    );
_betterPlayerController = BetterPlayerController(
      BetterPlayerConfiguration(
        fit: BoxFit.scaleDown,
        autoPlay: false,
        looping: false,
        allowedScreenSleep: false,
        fullScreenByDefault: false,
        autoDetectFullscreenDeviceOrientation: true,
        deviceOrientationsOnFullScreen: [DeviceOrientation.portraitUp],
        deviceOrientationsAfterFullScreen: [DeviceOrientation.portraitUp],
        controlsConfiguration: BetterPlayerControlsConfiguration(
          enableProgressBarDrag: false,
          enablePlaybackSpeed: false,
          enableSkips: false,
          enableMute: false,
          enableOverflowMenu: false,
        ),
      ),
      betterPlayerDataSource: betterPlayerDataSource,
    );
````

**Expected behavior**
Portrait video opening up in fullscreen.

**Screenshots**
![Simulator Screen Shot - iPhone 12 - 2021-06-10 at 17 11 28](https://user-images.githubusercontent.com/43028809/121551069-9a7aee80-ca0f-11eb-8a55-93398eaf1ef1.png)

**Flutter doctor**
[✓] Flutter (Channel stable, 2.2.1, on macOS 11.3 20E232 darwin-x64, locale en-DE)
[!] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    ✗ Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/docs/get-started/install/macos#android-setup for more details.
[✓] Xcode - develop for iOS and macOS
[✓] Chrome - develop for the web
[✓] Android Studio (version 4.1)
[✓] IntelliJ IDEA Community Edition (version 2020.2)
[✓] VS Code (version 1.56.2)
[✓] Connected device (3 available)

! Doctor found issues in 1 category.

**Better Player version**
- Version:^0.0.69

**Smartphone (please complete the following information):**
 - Device: iPhone12
 - OS: ios14

**Comment by CHENHUI-XU** [comment]
For iOS, the device does not check for autoDetectFullscreenDeviceOrientation. Is this correct? :) 

````
if (isAndroid) {
      if (_betterPlayerConfiguration.autoDetectFullscreenDeviceOrientation ==
          true) {
        final aspectRatio =
            widget.controller.videoPlayerController?.value.aspectRatio ?? 1.0;
        List<DeviceOrientation> deviceOrientations;
        if (aspectRatio < 1.0) {
          deviceOrientations = [
            DeviceOrientation.portraitUp,
            DeviceOrientation.portraitDown
          ];
        } else {
          deviceOrientations = [
            DeviceOrientation.landscapeLeft,
            DeviceOrientation.landscapeRight
          ];
        }
        await SystemChrome.setPreferredOrientations(deviceOrientations);
      } else {
        await SystemChrome.setPreferredOrientations(
          widget.controller.betterPlayerConfiguration
              .deviceOrientationsOnFullScreen,
        );
      }
    } else {
      await SystemChrome.setPreferredOrientations(
        widget.controller.betterPlayerConfiguration
            .deviceOrientationsOnFullScreen,
      );
    }
````

**Comment by jhomlala** [comment]
Fixed in 0.0.70.
